patient/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from customer.models import Customer
+
+class Patient(models.Model):
+    """
+        Patient model = Patient information storage
+
+        Arguments:
+            user_id	Number
+            first_name	String
+            last_name	String
+            first_name_pinyin	String
+            last_name_pinyin	String
+            gender	String
+            birthdate	DateTime
+            relationship	String
+            passport_number	String
+
+            note <optional>	String
+    """
+    # user_id = models.OneToOneField(Customer, on_delete=models.CASCADE)
+    user_id = models.CharField(max_length=100)

```

Comment:
I think `user_id` is a `ForeignKeyField` instead of `CharField`. `OneToOneField` might not be necessary, but I don't quite get why using `CharField`.

Reply:
+1